src/serverdlg.cpp

```diff
@@ -682,14 +682,15 @@ void CServerDlg::OnCLVersionAndOSReceived ( CHostAddress, COSUtil::EOpSystemType
 
 void CServerDlg::OnTimer()
 {
-    CVector<CHostAddress> vecHostAddresses;
-    CVector<QString>      vecsName;
-    CVector<int>          veciJitBufNumFrames;
-    CVector<int>          veciNetwFrameSizeFact;
+    CVector<CHostAddress>     vecHostAddresses;
+    CVector<QString>          vecsName;
+    CVector<int>              veciJitBufNumFrames;
+    CVector<int>              veciNetwFrameSizeFact;
+    CVector<CChannelCoreInfo> vecChanInfo;
 
     ListViewMutex.lock();
     {
-        pServer->GetConCliParam ( vecHostAddresses, vecsName, veciJitBufNumFrames, veciNetwFrameSizeFact );
+        pServer->GetConCliParam ( vecHostAddresses, vecsName, veciJitBufNumFrames, veciNetwFrameSizeFact, vecChanInfo );
```

Comment:
What does the Graphical user interface want to do this for?

Reply:
While not my code, the return values of GetConCliParam are used to populate the connection box at the top of the server dialog.  This method call was already here;  I just added the additional parameter.   This _exact_ discussion was had in another thread that I cannot find now -- but will keep looking.

Reply:
I'm not keen on unused variables (neither is the IDE).  If there are two different use cases, there should be two supporting methods.  One might call the other with a dummy parameter - or pass a dummy value here and ignore it in `GetConCliParam`.  At the moment, extra work is being done unnecessarily.